Playmate/Utilities/Constants.m

```diff
@@ -365,4 +387,30 @@ + (NSDictionary *)titleAttributes {
     return titleAttributes;
 }
 
+// for home view
++ (NSString *)getImageNameForSport:(NSString *)sport {
+    NSArray *imageNames = [NSArray arrayWithObjects:@"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"confetti",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",
+                                                    @"playmate_logo_transparent",nil];
+    
+    NSDictionary *sportToImageName = [[NSDictionary alloc] initWithObjects:imageNames forKeys:[Constants sportsList:NO]];
```

Comment:
`sportList` could be named `sportListIncludingAll:`

So, when you call the method, you know what is the boolean for.